### ISSUE TITLE
Audio source priority?

### ISSUE THREAD

**Issue by wuniversales** [KRTirtho/spotube#1958]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

Some songs do not play.

### Suggested solution

Instead skip to the next song if the audio is not found.
Why not allow to set priorities?

For example, 
First use jiosaavn, then youtube and then Piped.
This would avoid youtube blocking somehow.
Regards

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by KRTirtho** [comment]
Yeah it is already implemented. A fallback audio source is used chronologically.

So the order for different source's fallback as follows:
YouTube: Piped > JioSaavn
Piped: YouTube > JioSaavn
JioSaavn: YouTube > Piped

Currently, there's no way for user to customize it as it'll be too difficult to implement as a GUI. Also, JioSaavn is nowadays geo-locked so like Spotify they also now shows songs based geolocation and allowed Marketplaces.

Piped can be broken time to time due to YouTube API changes (takes days/weeks to fix). Same goes for YouTube.
